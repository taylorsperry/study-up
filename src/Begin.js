import React from "react"

const Begin = props => {
    return (
       <section className="quiz-card">
        <h2 className="correct">Welcome!</h2>
        <p className="explanation">StudyUp is a tool for practicing JavaScript fundamentals.</p>
        <button className="keep-practicing" onClick={props.toggleDisplay}>Let's get started</button>
       </section> 
    )
}
export default Begin