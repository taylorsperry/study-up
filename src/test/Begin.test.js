import React from "react";
import Begin from "../Begin";
import { shallow } from "enzyme"

describe("Begin", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Begin />)
  })

  it("should match snapshot when all data is passed correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
})