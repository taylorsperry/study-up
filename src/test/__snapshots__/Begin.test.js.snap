// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Begin should match snapshot when all data is passed correctly 1`] = `
<section
  className="quiz-card"
>
  <h2
    className="correct"
  >
    Welcome!
  </h2>
  <p
    className="explanation"
  >
    StudyUp is a tool for practicing JavaScript fundamentals.
  </p>
  <button
    className="keep-practicing"
  >
    Let's get started
  </button>
</section>
`;
